services/API-service/src/api/notification/notification-content/notification-content.service.ts

```diff
@@ -261,12 +266,22 @@ export class NotificationContentService {
     });
   }
 
-  private getTotalAffected(triggeredAreas: TriggeredArea[]) {
-    return parseFloat(
-      triggeredAreas
-        .reduce((acc, triggeredArea) => acc + triggeredArea.actionsValue, 0)
-        .toFixed(2),
+  private getTotal(
+    triggeredAreas: TriggeredArea[],
+    numberFormat: NumberFormat,
+  ) {
+    const total = triggeredAreas.reduce(
+      (acc, { actionsValue }) => acc + actionsValue,
+      0,
     );
+
+    if (numberFormat === NumberFormat.perc) {
+      // return average for percentage
+      return total / triggeredAreas.length;
```

Comment:
@gulfaraz this is not fully correct. You have to take the weighted average, in this case through 'weightVar=total_houses', which is also an indicator-attribute. In aggregates.service in the front-end there is an example of how this is used. You can maybe argue this is already to the sum we had before, but this will lead to differing numbers between email and portal.

Reply:
Please realize that having this total figure in the email is a new feature, that is not currently not on production. So we're losing nothing with just omitting it. 

Pseudocode formula is just something like: total-% = (area1-%*area1-total_houses + area2-%*area2-total_houses) / (area1-total_houses = area2-total_houses).

You probably don't have this additional data on 'total_houses' per area readily available at the place where you need it.

Reply:
> To be honest I was actually meaning to only omit the line in the percentage case.

This wasn't clear before and is now perplexing. Did the feature requirements describe this exception?

> Because it does work correctly in the sum-case, which is the vast majority of all implementations (population_affected).

We [agreed to have shared behaviour across countries and hazards](https://teams.microsoft.com/l/message/19:ad7e3a0c71ad4e9ba5b84a799eb507db@thread.v2/1728055901400?context=%7B%22contextType%22%3A%22chat%22%7D) without exceptions like these. I understand we implemented this feature before our agreement, but we're deciding to sign off on the feature now.

The requirement is to show the same aggregate value in the email as in the dashboard. If the suggestion is to show a value in the dashboard but not in the email, _I advise to not ship a partial implementation_. Let's refine this feature when it returns as feedback.

> But in this particular case, it might be better to split it with a clear comment in the code explaining why.

This addresses the discrepancy for us devs. However, it does not explain why the value is missing to the end-user, those demoing IBF, design, and other stakeholders.

We have the opportunity to,
1. Avoid the effort to document this exception
2. Avoid communicating this exception to the relevant people mentioned above
3. Avoid a future refactor to remove this exception

Reply:
Ok, I'm convinced by your argumentation, but do not appreciate the unfriendly tone of the choice of the word 'perplexing' (unless I'm misinterpreting somehow).

Reply:
By perplexing, I meant **confusing**. I apologize if it came across as unfriendly. It was very neutral.

Reply:
Ok, clear, thanks :)